### DISCUSSION TITLE
Architecture

### DISCUSSION THREAD

**Discussion by JohanCSmit** [COS301-SE-2022/Exploring-Self-Sovereign-Identity#95] [General]
@RebeccaPillay , could we have a meeting to discuss the changes to our diagram please? Whenever you are able to will be great!

**Comment by RebeccaPillay** [comment]
I can do 10h00. Does that work for you?

**Reply by JohanCSmit** [reply]
Perfect thanks.

**Reply by JohanCSmit** [reply]
@RebeccaPillay I'm on Discord as soon as you're ready.

**Comment by RebeccaPillay** [comment]
New Architectural Requirements Uploaded on Discord @JohanCSmit
